radix-engine/src/system/system.rs

```diff
@@ -1215,6 +1143,42 @@ where
 
         Ok(enabled)
     }
+
+    fn get_module_object_info(
+        &mut self,
+        node_id: &NodeId,
+        object_module_id: ObjectModuleId,
+    ) -> Result<ObjectInfo, RuntimeError> {
```

Comment:
The abstraction doesn't feel right - "module of a global object is not an object".